### ISSUE TITLE
Trivial: print "No free event buffers" warning to stderr instead of stdout

### ISSUE THREAD

**Issue by JohnPJenkins** [carothersc/ROSS#22]
tw-sched.c, line 180

**Comment by laprej** [comment]
Is this different enough from 719d8e48c2e988e9548b934feafc24e428485c51 to make it worthwhile?  Are you running into a situation where that logic is not being tripped but you're still reaching a problem state?

**Comment by JohnPJenkins** [comment]
I simply want the message that's currently printed to go to stderr rather than stdout (ie `fprintf(stderr, ...)` rather than printf(...)`). In many of my experiments I redirect stdout and stderr to different files to make it easier to grep through the resulting output.

Relatedly, I would love for there to be an option to kill the simulation after N consecutive failed "allocation" attempts. E.g., last evening before I left I set up a sweep of ~700 simulations, the 10th of which entered an infinite "buffers not available" loop I then killed this morning. A "`g_tw_max_consec_nobuf_ct`" or something (defaulting to `ULONG_MAX` or similar) would do the trick, along with a bit of extra logic to track and reset the failure count accordingly. Basically, an "opt-in" workaround for issue #6.
